docs/JLBP-0021.md

```diff
@@ -0,0 +1,28 @@
+---
+jlbp:
+  id: JLBP-21
+permalink: /JLBP-21
+---
+# Upload artifacts to Maven Central
+
+Developers expect to find artifacts in [Maven Central](https://repo1.maven.org/maven2/). All artifacts
+intended for public consumption
+should be published in the [Maven Central repository maintained by Sonatype](https://search.maven.org). Even if a project is not built with Maven,
+its artifacts can and should be published to Maven Central.
+
+From here the artifacts are mirrored to many different local and public repositories. 
+
+While Maven supports additional remote repositories,
```

Comment:
Clarify "additional remote repositories" and "third party repositories".

Reply:
done